Sources/iOSClientExposurePlayback/Events/DeviceInfo.swift

```diff
@@ -37,11 +37,13 @@ internal struct DeviceInfo {
     
     internal let appName: String?
     
+    internal let appVersion: String?
+    
     internal var cdnInfo: CDNInfoFromEntitlement?
     
     internal var analyticsInfo: AnalyticsFromEntitlement?
     
-    internal init(timestamp: Int64, connection: String, type: String? = nil, tech: String, techVersion: String,  cdnInfo: CDNInfoFromEntitlement? = nil , analyticsInfo: AnalyticsFromEntitlement? = nil, appName: String? = nil ) {
+    internal init(timestamp: Int64, connection: String, type: String? = nil, tech: String, techVersion: String,  cdnInfo: CDNInfoFromEntitlement? = nil , analyticsInfo: AnalyticsFromEntitlement? = nil, appName: String? = nil, appVersion: String? = nil) {
         self.timestamp = timestamp
         self.connection = connection
         self.type = type
```

Comment:
what is `type` here? and where is `appType`? 
You seem to always include these properties in other events, but not all of the in this event (DeviceInfo):
```
        var json: [String: Any] = [
            JSONKeys.eventType.rawValue: eventType,
            JSONKeys.timestamp.rawValue: timestamp,
            JSONKeys.connection.rawValue: connection,
            JSONKeys.player.rawValue: player,
            JSONKeys.deviceId.rawValue: device.deviceId,
            JSONKeys.deviceModel.rawValue: device.model,
            JSONKeys.os.rawValue: device.os,
            JSONKeys.appType.rawValue: "app",
            JSONKeys.osVersion.rawValue: device.osVersion,
            JSONKeys.manufacturer.rawValue: device.manufacturer,
            JSONKeys.height.rawValue: device.height,
            JSONKeys.width.rawValue: device.width
        ]
```

Reply:
Type is for `Airplay`

```
internal let type: String?` 
/// Optional string indicating the nature of playback from this device. Can be used to indicate this is an `Airplay` session : ´Airplay´
```

Reply:
Yes, I remember now as we discussed it on slack. It was removed in the new specification, but leave it for now to keep this PR isolated to the jira ticket.